ycms/cms/templates/_raw.html

```diff
@@ -43,7 +43,7 @@
     <body class="font-default text-lg bg-background-50">
         {% if messages and not messages_suppressed %}
             <div id="messages"
-                 class="absolute top-32 right-8 flex flex-col max-w-screen-sm z-50">
+                 class="absolute top-8 right-8 flex flex-col max-w-screen-sm z-50">
```

Comment:
> But I already set top and right like follows, I did't see any different after I add `md:top-8 md:right-8`.
> 
> ```
>             <div id="messages"
>                  class="absolute top-8 right-8 flex flex-col max-w-screen-sm z-50">
> ```

```suggestion
                 class="absolute md:top-8 md:right-8 flex flex-col max-w-screen-sm z-50">
```

Without: 
![image](https://github.com/charludo/ycms/assets/47758554/b1e0709c-422c-4844-b175-c3a0061e6053)

With:
![image](https://github.com/charludo/ycms/assets/47758554/7df12784-6386-42b9-9399-d4292bbb933f)